### DISCUSSION TITLE
Event "object:scaled" missing?

### DISCUSSION THREAD

**Discussion by maniac-0s** [fabricjs/fabric.js#9091] [Q&A] [answered]
I once wrote a fabric app like over 2 years ago. For a new project I want to reactivate the app and are currently walking through all issues to make it working with the latest versions of node and fabric.

In my old project I used at some point the event `canvas.on("object:scaled", handleScaled);` but that does not seem to do anything anymore. I the function `handleScaled` would be called if I use` "object:modified"` at this point but since there are many possibilities an object got modified and I wanted to call `handleScaled` **only** when the scale operation was finished (so I can't use `"object:scaling"` as that event is raised even more often), I'd like to ask how I can do this when there is no "object:scaled" event anymore.

**Comment by melchiar** [comment, marked as answer]
Hi @maniac-0s, `object:scaled` and other events were removed in v5. Use `object:modified` instead, then check the `action` property of the event to filter for scaled.

See http://fabricjs.com/v5-breaking-changes
